lahja/endpoint.py

```diff
@@ -376,7 +376,9 @@ def stop(self) -> None:
         self.ipc_path.unlink()
 
     @asynccontextmanager  # type: ignore
-    async def run(self) -> AsyncIterable['Endpoint']:
+    async def run(self) -> AsyncIterable['Endpoint']:  # type: ignore
```

Comment:
Shouldn't this use the `check_event_loop` decorator that all the other methods also use?

Reply:
Something about the `asynccontextmanager` didn't play well with with it.